tools/update_test_data.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Import utility functions
+# shellcheck source=./tools/_functions.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
+
+require_installed
+require_database
+
+deescalate_privileges integreat-cms-cli dumpdata --natural-primary --natural-foreign --exclude auth --exclude contenttypes --o /integreat_cms/cms/fixtures/test_data.json
```

Comment:
```suggestion
deescalate_privileges integreat-cms-cli dumpdata --natural-primary --natural-foreign --exclude auth --exclude contenttypes --o /integreat_cms/cms/fixtures/test_data.json
# Format the new function
source "$(dirname "${BASH_SOURCE[0]}")/prettier.sh"
```
By running prettier we can make sure that the generated json file is consistently formatted

Reply:
+1 for prettier, but it's probably even nicer to just call `prettier` directly on the one changed file?